runner/proto/policy_behavior.go

```
@@ -0,0 +1,108 @@
+		PolicyPaths:    slices.Clone(r.PolicyPaths),
+		ApiServer:      r.ApiServer,
+		PolicyBehavior: mergePolicyBehavior(defaults, r.PolicyBehavior),
+	}
+}
+
+func (r *EvalRequest) WithUndefinedMappedTo(behavior []string) *EvalRequest {
+	if r == nil {
+		return nil
+	}
+
+	copy := &EvalRequest{
```

golangci-lint (2.12.2)

[error] variable copy has same name as predeclared identifier

(predeclared)